modules/ingester/trace.go

```diff
@@ -34,7 +34,7 @@ func (t *trace) Push(_ context.Context, trace []byte) error {
 	if t.maxBytes != 0 {
 		reqSize := len(trace)
 		if t.currentBytes+reqSize > t.maxBytes {
-			return status.Errorf(codes.FailedPrecondition, "%s max size of trace (%d) exceeded while adding %d bytes", overrides.ErrorPrefixTraceTooLarge, t.maxBytes, reqSize)
+			return status.Errorf(codes.FailedPrecondition, "%s max size of trace (%d) exceeded while adding %d bytes, the traceID is %s", overrides.ErrorPrefixTraceTooLarge, t.maxBytes, reqSize, string(t.traceID))
```

Comment:
nit: %s max size of trace (%d) exceeded while adding %d bytes to trace %s

Also please use `hex.EncodeToString()` to translate the byte slice into a hex encoded string.

Reply:
> Thanks for the first time contribution! There are a few small changes requested.
> 
> Also, can you add a changelog entry?

Thank you so much for the review, I will definitely do that!

Reply:
> nit: %s max size of trace (%d) exceeded while adding %d bytes to trace %s
> 
> Also please use `hex.EncodeToString()` to translate the byte slice into a hex encoded string.

Thanks a lot for the suggestion, will definitely fix it :smile:

Reply:
Fixed :smile: